### ISSUE TITLE
"title" prop no longer renders screen-reader-accessible content

### ISSUE THREAD

**Issue by phb-ig** [FortAwesome/react-fontawesome#586]
**Describe the bug**
This code:
<img width="625" height="91" alt="Image" src="https://github.com/user-attachments/assets/420c8d31-d324-4886-ae22-54786c27cc7b" />

Should generate this HTML:
<img width="849" height="144" alt="Image" src="https://github.com/user-attachments/assets/8a43f59f-e165-4d8c-b430-43fea03dc372" />

But instead generates this:
<img width="828" height="128" alt="Image" src="https://github.com/user-attachments/assets/8b838f3c-e8e5-4ade-baa1-2b8c3993816f" />

This appears to be a bug with how `react-fontawesome` is integrating with the v7+ icon libraries.

The code is generated properly under
```
"@fortawesome/fontawesome-svg-core": "6.7.2",
"@fortawesome/free-regular-svg-icons": "6.7.2",
"@fortawesome/free-solid-svg-icons": "6.7.2",
"@fortawesome/react-fontawesome": "3.0.0",
```

But fails to generate the label with
```
"@fortawesome/fontawesome-svg-core": "7.0.0",
"@fortawesome/free-regular-svg-icons": "7.0.0",
"@fortawesome/free-solid-svg-icons": "7.0.0",
"@fortawesome/react-fontawesome": "3.0.0",
```

**Comment by phb-ig** [comment]
**CodeSandbox examples**

Working: https://codesandbox.io/p/sandbox/fa-working-title-fwlj9j
Broken: https://codesandbox.io/p/sandbox/fa-broken-title-svjh67

**Comment by phb-ig** [comment]
I'm using it as semantic, yeah. The change makes sense to keep a tooltip separate from button content :)

I wasn't able to find this change in the main `fontawesome` release notes, or those for `react-fontawesome`. Is it possible to mention it more prominently, since this feature is important in maintaining compliance with WCAG standards?

Also: I saw in the release notes for `react-fontawesome` that `aria-hidden="false"` should be automatically set when `aria-label` is nonempty, however that appears to not be working properly: https://codesandbox.io/p/sandbox/hopeful-poitras-p8xdfy?file=%2Fsrc%2FApp.js

**Comment by charles4221** [comment]
Hi @phb-ig, while it may not have been in the release notes explicitly due to v7 being a very large release, the Accessibility documentation has been updated here providing examples using `aria-label` rather than `title`:

https://docs.fontawesome.com/web/dig-deeper/accessibility#making-semantic-icons-accessible

> aria-hidden="false" should be automatically set when aria-label is nonempty, however that appears to not be working properly

Thanks for noting this, it was definitely working on my end but I will dig further into this and make sure we get some unit tests in place to ensure it works correctly.
